ci/sunspot_test_script.sh

```diff
@@ -13,7 +13,13 @@ start_solr_server() {
   # go to sunspot_solr folder and install dependencies
   current_path=`pwd`
   cd ../sunspot_solr
-  bundle install --quiet --path vendor/bundle
+
+  if [ "$(printf "%s\n" "3.2" "$RUBY_VERSION" | sort -V | tail -n1)" = "$RUBY_VERSION" ]; then
+    bundle config set --local path 'vendor/bundle'
```

Comment:
@s4na can you explain the purpose of this conditional? would `bundle config set` + `bundle install` work the same way for all ruby versions?

Reply:
if this works the same way, then the conditional can be removed, right?